webapp/src/styles/ResourceMetadata.scss

```diff
@@ -43,6 +43,21 @@
       float: right;
       color: #2586a6;
     }
+
+    .info-link {
+      color: #2586a6;
```

Comment:
Is this a color already used elsewhere? Is it available in `variables.scss`? (If not, could these go in there?)